project-Employee/frontend/src/App.tsx

```diff
@@ -27,13 +22,15 @@ const [url,setUrl] = useState()
             .catch((error)=> {console.error(error)})
     }
     function addEmployee(newEmployee: NewEmployee) {
-        axios.post("/api/employees",newEmployee)
+            const data = new FormData()
+            data.append("data",new Blob([JSON.stringify(newEmployee)],{'type':"application/json"}))
+        axios.post("/api/employees",data)
             .then((addEmployeeResponse) =>{
                 setEmployees([addEmployeeResponse.data,...employees])
             })
             .catch(console.error)
     }
-    function updateEmployee(employee: employee) : void{
+    function updateEmployee(employee: employee) {
```

Comment:
du könntest deine funktionen mit axios Requests auch auslagern in hooks (z.B. useEmployee) und hier in die App über const {addEmployee, deleteEmployee} = useEmployee() reinholen.